scss/layout/_containers.scss

```diff
@@ -547,3 +552,51 @@
     border-top: 1px solid $color-line-border;
   }
 }
+
+.about-page-section {
+  grid-column: 1/-1;
+}
+
+.author {
+  &-cards-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+  
+    @media #{$tablet-bp} {
+      flex-direction: row;
+    }
+  }
+
+  &-card {
+    display: block;
+    margin: 3rem auto;
+    padding: 1.5rem;
+    max-width: 25rem;
+    border-radius: 1.5rem;
+    background: none;
+    transition: background 3s;
+  
+    @media #{$tablet-bp} {
+      margin: 1rem;
+      max-width: 100%
+    }
+  
+    @media #{$desktop-bp} {
+      margin: 2rem;
+      width: 20rem;
+    }
+
+    &:hover {
+      background: linear-gradient(to bottom, $color-main, $color-white);
+    }
+
+    &-content {
+      transition: opacity 0.8s 0s;
+    }
+
+    &:hover > &-content {
+      opacity: 1;
+    }
+  }
+}
```

Comment:
Please convert this scope to a BEM pattern.

Reply:
Code updated, also updated typography.scss file to match BEM pattern 
e9a8403655a942a7d2a1ed379419d8edc68802b7